ome_zarr/writer.py

```diff
@@ -0,0 +1,112 @@
+"""Image writer utility
+
+"""
+import json
+import logging
+from pathlib import Path
+from typing import Any, List, Tuple, Union
+
+import dask.array as da
+import numpy as np
+import zarr
+
+from .io import parse_url
+from .reader import Node
+from .types import JSONDict
+
+LOGGER = logging.getLogger("ome_zarr.writer")
+
+
+def write_image(
+    path: str,
+    image: np.ndarray,
+    name: str = "0",
+    group: str = None,
+    chunks: Union[Tuple[int], int] = None,
```

Comment:
If this method is to handle downsampling internally, I wonder if we want to pass multiple chunk sizes or keep them consistent.

Reply:
seems to me it's easier to keep the same chunks for all sample sizes. Or scale the number of chunks - maybe in the x, y dimensions - to have a consistent chunk size across scales? I'd assume the specified chunk size is for the 1:1 scale.

It sounds complicated to ask the user to input one chunk size for each scale - imho it should be as transparent as possible. 

Maybe it will get clearer when `write_multiscales` is integrated here?

Reply:
Yeah, I had gotten that far ;) I think you're right that it's more difficult than one would want in general. Is it worth though providing all the options, i.e.

```
Union[Tuple[Tuple[int]], Tuple[int], int]
```